worker/buildbot_worker/commands/transfer.py

```diff
@@ -107,7 +107,8 @@ def start(self):
         self._reactor.callLater(0, self._loop, d)
 
         def _close_ok(res):
-            self.fp = None
+            if self.fp:
+                self.fp.close()
```

Comment:
Can you also comment out these two lines from the .py3.notworking.txt file:

```
buildbot.test.integration.test_transfer.TransferStepsMasterNull.test_transfer
buildbot.test.integration.test_transfer.TransferStepsMasterPb.test_transfer
```

and add that to your review?